extensions/wikia/Wall/WallThread.class.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
 class WallThread {
+	const DATABASE_LIMIT = 500;
```

Comment:
Style-wise, I think this constant is fine; upper case and underscore separated.  I think it could probably be more descriptive though.  Since its a limit on the number of replies, maybe ```FETCHED_REPLY_LIMIT```?